lib/std/os/linux.zig

```diff
@@ -213,6 +214,21 @@ pub const MAP = switch (native_arch) {
         UNINITIALIZED: bool = false,
         _: u5 = 0,
     },
+    .loongarch64 => packed struct(u32) {
```

Comment:
This looks to be identical to the `riscv32`/`riscv64` prong above. Let's just merge them.

Reply:
Yes. It seems that the newly supported architectures in Linux kernel do not have arch-specific flags and they just use the generic ones.

- https://elixir.bootlin.com/linux/v5.19/source/include/uapi/asm-generic/mman.h
- https://elixir.bootlin.com/linux/v5.19/source/include/uapi/asm-generic/mman-common.h